embulk-core/src/main/java/org/embulk/exec/PreviewExecutor.java

```diff
@@ -36,6 +36,10 @@
     public interface PreviewTask
             extends Task
     {
+        @Config("exec")
+        @ConfigDefault("{}")
+        public ConfigSource getExecConfig();
```

Comment:
Sorry, I'm also not sure why the `PreviewTask` has `preview_sample_rows` option. We need to ask @frsyuki about the reason if needed. But, I think, it's better that the option is moved within `exec:` section (or system config) as written in TODO comment.